.github/workflows/test.yml

```
@@ -36,12 +36,23 @@ jobs:
         run: |
           mamba --version 2>&1 | tee test_linux_artifacts_python_${{ matrix.python-version }}/conda_version.txt
           python -V 2>&1 | tee test_linux_artifacts_python_${{ matrix.python-version }}/python_version.txt
+          echo "pver0=$(python -c 'import sys; print(sys.version_info[1])')" >> $GITHUB_ENV
```

```suggestion
          echo "pver0=$(python -V)" >> $GITHUB_ENV
```
would be a bit simpler, we wouldn't expect any of the version digits to change.

---

that might result in unexpected behaviour - `python -V` may return something like `Python 3.9.1` or `3.9.1` depending on what machine you are, also encoding spaces and comparing more complex strings than just a Number is prone to encoding issues etc, I'd leave it like this tbh

---

Well, it should return the exact same thing when called twice from the same installation on the same machine, right?

---

What if Julia alter it? Ok now we're getting real paranoid 😆 Commit the change and merge? 🍺